examples/tp/hello_tm.py

```diff
@@ -6,7 +6,7 @@
 # following terms and conditions apply:
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero Public License version 3 as
+# it under the terms of the GNU General Public License version 3 as
```

Comment:
Good catch Richard. @arhik Please be sure to revert these changes from GPL to AGPL.